src/domain/LottoPrize.js

```diff
@@ -0,0 +1,22 @@
+export default class LottoPrize {
+  requiredMatchCount;
+  bonusMatched;
+  prizeMoney;
+  matchCount = 0;
+
+  constructor(requiredMatchCount, bonusMatched, prizeMoney) {
```

Comment:
여러 개의 인자를 받을 때는 객체로 인자를 받는 것도 고려해보시면 좋을거 같아요!

```suggestion
  constructor({ requiredMatchCount, bonusMatched, prizeMoney }) {
```